pkg/reconciler/v1alpha1/eventlistener/eventlistener.go

```diff
@@ -396,7 +359,7 @@ func (r *Reconciler) reconcileDeployment(ctx context.Context, logger *zap.Sugare
 	return nil
 }
 
-func getDeployment(el *v1alpha1.EventListener) *appsv1.Deployment {
+func (r *Reconciler) getDeployment(el *v1alpha1.EventListener) *appsv1.Deployment {
```

Comment:
If we make this `getDeployment(el *v1alpha1.EventListener, config Config)` the dependency on the `Config` is clearer and the function is a bit easier to test (we don't have to setup an entire reconciler struct)